core/sawtooth/client.py

```diff
@@ -54,8 +54,8 @@ def _sign_message_with_transaction(transaction, message_type, key):
     of a sha256 hexdigest.
     """
     transaction['Nonce'] = time.time()
```

Comment:
I think most of this directory was removed in the great purge of 69cbb. By the comment, remainder was left for future work to port to 0.8. https://github.com/hyperledger/sawtooth-core/commit/69cbb589febd5ab451cd9b8dd54dabe728f84ad4